.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
Was this provided by the angular cli upgrade too or maybe because you're using yarn 2?